src/ide/spans.ts

```diff
@@ -610,15 +610,25 @@ export class CodeBlockSpan extends BlockSpan {
   protected widgetLine:number;
   protected widget:CodeMirror.LineWidget;
   protected widgetElem:HTMLElement;
+  protected languageLabelElem:HTMLElement;
   protected enableToggleElem:HTMLElement;
 
   protected footerWidgetLine:number;
   protected footerWidget:CodeMirror.LineWidget;
   protected footerWidgetElem:HTMLElement;
 
+  syntax() : string {
+    return this.source.info ? this.source.info.toLowerCase().split(" ")[0] : "eve";
+  }
+
+  syntaxHighlight() : string {
+    // provide codemirror syntax highlight indications for css blocks only
+    return this.syntax() === "css" ? "cm-s-default" : "";
```

Comment:
We should look into using CodeMirror's overlay mode to enable hypothetical other languages in the future [1]. This is a want right now, not a need. Take a look to see how long you think it would take. If it seems significant we can instead drop an `// @TODO: ...` in the code and revisit it when it becomes relevant.

[1] https://codemirror.net/demo/mustache.html
[2] https://codemirror.net/addon/mode/overlay.js